docs/parameters_desc.md

```diff
@@ -0,0 +1,189 @@
+This document attempts to describe the important parameters in Macrobase that
+affect its behavior.
+
+We first describe workload-specific parameters, that _need_ to be specified for
+every workload. These parameters do not have default values, so Macrobase will
+throw an exception if these aren't specified.
+
+<table class="table">
+<tr><th>Name</th><th>Default</th><th>Meaning</th></tr>
+<tr>
+  <td><code>macrobase.query.name</code></td>
+  <td>(none)</td>
+  <td>
+  Human-readable task name, used to identify the workload.
+  </td>
+</tr>
+<tr>
+  <td><code>macrobase.loader.attributes</code></td>
+  <td>(none)</td>
+  <td>
+  List of attributes to group by. These
```

Comment:
We should probably come up with a better name for this. Summarization attribute?